app/controllers/recipes_controller.rb

```diff
@@ -0,0 +1,16 @@
+class RecipesController < ApplicationController
+
+  def new; end #root
+
+  def index
+    recipes = EdamamApiWrapper.list_recipes(params[:query])
+
+    @recipes = Kaminari.paginate_array(recipes).page(params[:page]).per(12)
```

Comment:
If you expect the lib file to raise an error, you should wrap this call in a `begin`/`rescue`.